README.md

```diff
@@ -48,21 +48,31 @@ Note that Vroom requires a version of vim built with the `+clientserver`
 option (run `vim --version` to check).  See `:help clientserver` for
 additional requirements.
 
-The easiest way to install vroom is to cd into the vroom directory and run
+If you're on Ubuntu or Debian, you can install
+[release packages](https://github.com/google/vroom/releases) from GitHub.
+
+Otherwise, the easiest way to install vroom is to clone the vroom repository
+from Github, cd into the vroom directory, and run
 ```sh
 python3 setup.py build && sudo python3 setup.py install
 ```
 
-You can also install
-[release packages](https://github.com/google/vroom/releases) from github. To
```

Comment:
I don't think we need to get into how to build a deb yourself here. Maybe in more in-depth install/maintainer docs that are yet to be written.